src/CMakeLists.txt

```diff
@@ -103,3 +103,12 @@ install(TARGETS umf
 )
 
 add_subdirectory(pool)
+
+# TODO: enable proxy_lib on Windows
+if(LINUX)
+    if(UMF_BUILD_LIBUMF_POOL_JEMALLOC)
```

Comment:
how does it work with `PROXY_LIB_USES_SCALABLE_POOL` variable set in proxy_lib/CMakeLists.txt?

Reply:
Done